pyproject.toml

```diff
@@ -17,6 +17,8 @@ lines_after_imports = 2
 known_first_party = "equisolve"
 
 [tool.pytest.ini_options]
-addopts = "-p pytest_cov"
-python_files = ["test_*.py", "*_test.py", "tests.py"]
-testpaths = "tests"
+python_files = ["*.py"]
+testpaths = [
+    "tests/equisolve_tests/*.py",
+    "tests/equisolve_tests/**/*.py",
```

Comment:
Why are you placing tests here and not just as `tests/*.py`?

Reply:
I tried things and hoped that this changes something.